addons/wp/functions/Modules/fnc_moduleArtillery.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: jokoho482
+ * TODO
```

Comment:
Here is how I suggest this module work.

Module is put down with the arguments: 
-  SIDE
- MAIN SALVO  (default 6 rounds) 
- SPREAD (default 75m) 

Module now waits until side in question has a ready artillery piece. 
The function now engages as usual with the _leader of the artillery_ as the caller  (argument 2 in script)

Essentially the module is put down and the artillery self-designates a target once ready.  As an improvement, we _may_ want to consider adding an option to skip the check rounds.

Reply:
Essentially: 
1. Pick a side
2. Wait Until that side has available artillery
3. Leader of artillery spawns function: 

`[<artillery>, <position of module/logic>, <leader of artillery>] spawn lambs_wp_taskArtillery;`